nuxt.config.js

```diff
@@ -120,10 +120,10 @@ module.exports = {
     'cookie-universal-nuxt',
     ['nuxt-i18n', {
       locales: i18nLocales,
-      baseUrl: process.env.I18N_BASE_URL,
+      baseUrl: ({ store }) => store.getters['http/canonicalUrlWithoutLocale'],
```

Comment:
This needs to use the getter `http/origin`. Otherwise it results in hrefs like:
```html
<link data-n-head="ssr" data-hid="alternate-hreflang-bg" rel="alternate" href="http://localhost:3001/search?page=1&amp;view=grid&amp;query=/bg/search?page=1&amp;view=grid&amp;query=" hreflang="bg">
```